pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -211,6 +213,23 @@ func (cs *ConntrackConnectionStore) AddOrUpdateConn(conn *flowexporter.Connectio
 		existingConn.ReverseBytes = conn.ReverseBytes
 		existingConn.ReversePackets = conn.ReversePackets
 		existingConn.TCPState = conn.TCPState
+
+		prevIsActive := existingConn.PriorityQueueItem.IsActive
+		// Reset the pqItem idle expire time and set IsActive to be true when
+		// there are changes either in stats or TCP state.
+		if (conn.OriginalPackets > existingConn.PriorityQueueItem.PrevPackets) ||
+			(conn.ReversePackets > existingConn.PriorityQueueItem.PrevReversePackets) ||
+			(existingConn.PriorityQueueItem.Conn.TCPState != conn.TCPState) {
+			existingConn.PriorityQueueItem.IsActive = true
```

Comment:
This is not required right. If the corresponding connection becomes active, then we anyway add new pqItem replacing the old one. Same with updating idle expire time.

Reply:
Trying moving this block inside the 
```
if prevIsActive{
      check activeness
 } else {
      check activeness
}
```